wsimod/nodes/land.py

```diff
@@ -1199,7 +1295,85 @@ def __init__(
                 self.nutrient_pool.fast_pool.storage["P"] = initial_soil_storage[
                     "org-phosphorus"
                 ]
+    
+    def infer_sow_harvest_calendar(self):
+        """Infer basic sow/harvest calendar and indicate autumn-sown.
+        Returns:
+            (list): havest/sow calendar
+            (list): ground cover stages
+            (list): crop cover stages
+            (boolean): indication for autumn-sown crops
+        """
+        # Infer basic sow/harvest calendar
+        harvest_sow_calendar = [
+            0,
+            self.sowing_day,
+            self.harvest_day,
+            self.harvest_day + 1,
+            365,
+        ]
+        ground_cover_stages = [0, 0, self.ground_cover_max, 0, 0]
+        crop_cover_stages = [0, 0, self.crop_cover_max, 0, 0]
 
+        # Use day number of 181 to indicate autumn-sown (from HYPE)
+        if self.sowing_day > 181:
+            autumn_sow = True
+        else:
+            autumn_sow = False
+        
+        return harvest_sow_calendar, ground_cover_stages, crop_cover_stages, autumn_sow
+    
+    def calculate_available_water(self):
+        """Calculate total/readily available water based on capacity/wp.
+        Returns:
+            (float): total available water
+            (float): readily available water
+        """
+        # Calculate parameters based on capacity/wp
+        total_available_water = self.field_capacity_m - self.wilting_point_m

```

Comment:
Should TAW/RAW just be a `@property` maybe?

Reply:
#95